activemq-web-console/src/main/webapp/WEB-INF/activemq.xml

```diff
@@ -34,6 +34,9 @@
     <transportConnectors>
       <transportConnector name="openwire" uri="tcp://localhost:61616" />
       <transportConnector name="stomp" uri="stomp://localhost:61613" />
+      <!-- Add Jetty Transport Connector for Web Console -->
```

Comment:
That's not correct: the jetty transport is not for the webconsole. Jetty transport is for http transport connect.
The WebConsole is managed outside of the broker by inclusion of `jetty.xml`.